benchexec/tools/WitForJBMC.py

```diff
@@ -0,0 +1,32 @@
+import benchexec.tools.template
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+ def executable(self, tool_locator):
+  return tool_locator.find_executable("Wit4JBMC.py")
+ def name(self):
+  return "Wit4JBMC"
+ def cmdline(self, executable, options, task, rlimits):
+  return [executable] + options + list(task.input_files_or_identifier)
```

Comment:
Does your tool support tasks without any input file? If not, it would be better to use `task.input_files` because this will produce a nice error message for the user in case of a task without input files.